Database/compiler.lua

```diff
@@ -748,32 +753,27 @@ function QuestieDBCompiler:CompileTableTicking(tbl, types, order, lookup, after,
     QuestieDBCompiler.pointerMap = {}
     QuestieDBCompiler.stream = QuestieStream:GetStream("raw")
 
-    QuestieDBCompiler.ticker = C_Timer.NewTicker(0.01, function()
+    while true do
+        coroutine.yield()
         for _=0,Questie.db.global.debugEnabled and 4000 or (entriesPerTick or 48) do
```

Comment:
There are more magic numbers in there. Similar to the ones in Database/Corrections/QuestieCorrections.lua